charts/merlin/templates/mlflow-service-account.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if and .Values.mlflow.serviceAccount.create }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: {{ default "mlflow" .Values.mlflow.serviceAccount.name }}
+    namespace: {{ .Release.Namespace }}
+    annotations:
+      {{- toYaml .Values.mlflow.serviceAccount.annotations | nindent 6 }}
```

Comment:
Might be better to set this if `annotations` is not empty

Reply:
Added checks